src/iam/grpchelper.cpp

```diff
@@ -0,0 +1,180 @@
+/*
```

Comment:
`grpchelpers` should be in utils. No need for new module for them.

Reply:
No, there should be linear modules topology such as:

```
config
database
client
server
```

grpc should be available for all modules and common part should be moved to utils.